vendors/cypress/boards/CYW943907AEVAL1F/aws_tests/application_code/main.c

```diff
@@ -145,15 +145,17 @@ void vApplicationDaemonTaskStartupHook( void )
      * running, here. */
     wiced_init();
 
-    /* Initialize the AWS Libraries system. */
-    if( SYSTEM_Init() == pdPASS )
-    {
-        /* Connect to the Wi-Fi before running the tests. */
-        prvWifiConnect();
+    /* Connect to the Wi-Fi before running the tests. */
+    prvWifiConnect();
```

Comment:
Cypress is using lwIP, and not FreeRTOS+TCP? If so, that's good in theory that +TCP wouldn't be racing the RNG before trying to send its first packets.

However, it raises the question of how lwIP gets its transaction IDs, sequence, and port numbers. It looks like lwIP is initialized in wiced_init, above.